modules/errors.go

```diff
@@ -0,0 +1,12 @@
+package modules
+
+import "github.com/NebulousLabs/errors"
+
+// ErrHostFault is an error that is usually extended to indicate that an error
+// is the host's fault.
+var ErrHostFault = errors.New("")
```

Comment:
error string should be human readable. IIRC the `Extend` function will add some colons and brackets, so an empty error will look pretty confusing when printed out.

maybe say "host has returned an error:"

Reply:
lol I told myself I would worry about a meaningful string later. Apparently I forgot about that.